vignettes/estimate_infections_workflow.Rmd

```diff
@@ -258,14 +290,15 @@ All the options are combined in a call to the `estimate_infections()` function.
 For example, using some of the options described above one could call
 
 
-```r
+``` r
 def <- estimate_infections(
   example_confirmed, 
   generation_time = gt_opts(generation_time),
   delays = delay_opts(delay),
   rt = rt_opts(prior = rt_prior)
 )
-#> Warning: There were 3 divergent transitions after warmup. See
+#> DEBUG [2024-09-19 14:46:51] estimate_infections: Running in exact mode for 2000 samples (across 4 chains each with a warm up of 250 iterations each) and 147 time steps of which 7 are a forecast
```

Comment:
Was this rendered in debug mode? I think ideally run the `precompile.R` script in a pristine environment.

Reply:
> Was this rendered in debug mode? I think ideally run the `precompile.R` script in a pristine environment.

I precompiled it as usual in a fresh environment. I don't understand the debug mode question.

Reply:
There's a new DEBUG message which wasn't there previously

Reply:
This probably comes from rendering in an interactive session vs. using `Rscript` - the render actions will remove these again so it's fine.

https://github.com/epiforecasts/EpiNow2/blob/793525c0ce0b5b627a618191ff8fdf48df3710f6/R/epinow.R#L99